src/BinSkim.Rules/RuleResources.resx

```diff
@@ -266,43 +310,63 @@ Modules triggering this check were:
+To resolve this issue, don't set /NXCOMPAT:NO on link.exe command line and allow it to be enabled by default.</value>
   </data>
   <data name="BA2016_MarkImageAsNXCompatible_Description" xml:space="preserve">
-    <value>Binaries should be marked as NX compatible to help prevent execution of untrusted data as code. The NXCompat bit, also known as "Data Execution Prevention" (DEP) or "Execute Disable" (XD), triggers a processor security feature that allows a program to mark a piece of memory as non-executable. This helps mitigate memory corruption vulnerabilities by preventing an attacker from supplying direct shellcode in their exploit (because the exploit comes in the form of input data to the exploited program on a data segment, rather than on an executable code segment). Ensure that your tools are configured to mark your binaries as NX compatible, e.g. by passing /NXCOMPAT to the C/C++ linker.</value>
+    <value>Binaries should be marked as NX compatible to help prevent execution of untrusted data as code. The NXCompat bit, also known as "Data Execution Prevention" (DEP) or "Execute Disable" (XD), triggers a processor security feature that allows a program to mark a piece of memory as non-executable.
+This helps mitigate memory corruption vulnerabilities by preventing an attacker from supplying direct shellcode in their exploit (because the exploit comes in the form of input data to the exploited program on a data segment, rather than on an executable code segment).
+To resolve this issue, don't set /NXCOMPAT:NO on link.exe command line and allow it to be enabled by default.</value>
   </data>
   <data name="BA2016_Pass" xml:space="preserve">
     <value>'{0}' is marked as NX compatible, helping to prevent attackers from executing code that is injected into data segments.</value>
   </data>
   <data name="BA2018_EnableSafeSEH_Description" xml:space="preserve">
-    <value>X86 binaries should enable the SafeSEH mitigation to minimize exploitable memory corruption issues. SafeSEH makes it more difficult to exploit vulnerabilities that permit overwriting SEH control blocks on the stack, by verifying that the location to which a thrown SEH exception would jump is indeed defined as an exception handler in the source program (and not shellcode). To resolve this issue, supply the /SafeSEH flag on the linker command line. Note that you will need to configure your build system to supply this flag for x86 builds only, as the /SafeSEH flag is invalid when linking for ARM and x64.</value>
+    <value>X86 binaries should enable the SafeSEH mitigation to minimize exploitable memory corruption issues. SafeSEH makes it more difficult to exploit vulnerabilities that permit overwriting SEH control blocks on the stack, by verifying that the location to which a thrown SEH exception would jump is indeed defined as an exception handler in the source program (and not shellcode).
+To resolve this issue, supply the /SafeSEH flag on link.exe command line.
```

Comment:
>link.exe command line [](http://example.com/codeflow?start=51&length=21)

should we change all instance of `link.exe` in the PR to just `linker`.
For user that is not using Microsoft toolchain.

Reply:
Most of the switches are tool specific, clang/gcc have their own, not /SafeSEH.

Reply:
/SafeSEH is only applicable to link.exe, clang/GCC have different switches.

Reply:
Yes. just a note not related to this PR, imo we have assumptions in a lot places the user is using Microsoft toolchain, while BinSkim target is just PE files.